zerver/lib/hotspots.py

```diff
@@ -43,11 +43,21 @@
     },
 }
 
+
+NON_INTRO_HOTSPOTS: Dict[str, Dict[str, Promise]] = {}
+
 # We would most likely implement new hotspots in the future that aren't
 # a part of the initial tutorial. To that end, classifying them into
 # categories which are aggregated in ALL_HOTSPOTS, seems like a good start.
 ALL_HOTSPOTS: Dict[str, Dict[str, Promise]] = {
-    **INTRO_HOTSPOTS,
+    **{
+        hotspot_name: {**INTRO_HOTSPOTS[hotspot_name], "has_trigger": False}
```

Comment:
I have also updated the `ALL_HOTSPOTS` list so it can display `NON_INTRO_HOTSPOTS` as it's expected.